rc/filetype/crystal.kak

```diff
@@ -239,6 +250,14 @@ define-command -hidden crystal-fetch-keywords %{
     }
 }
 
+# https://crystal-lang.org/api/toplevel.html#macro-detail
+define-command -hidden crystal-fetch-macros %{
+    set-register dquote %sh{
+        curl --location https://crystal-lang.org/api/toplevel.html |
+        kak -f '%1smacro\h+<lt>strong<gt>(\w+[?!]?)<lt>/strong<gt><ret>y%<a-R>a<ret><esc><a-_>a<del><esc>'
```

Comment:
As mentioned in #3523, assuming `curl` is on the system, fetching arbitrary data and running `kak` on top is probably not the play.

Reply:
It is just a command for the maintainer to see the updates.

Reply:
I'd like those to be removed from the .kak file, its a useful snippet for you, but it is not really something we need to keep around for all users, and it also depends on how that webpage is formatted which might change.

Reply:
If someone else want to maintain the language he has the helpers to be sure to not miss any keyword; if the command fails he will see it in its clipboard and adapts the command.

Reply:
https://crystal-lang.org/api/toplevel.html is a trust worthy URL at this point but there no guarantee what content it may serve in the future. Piping the contents of the URL to `kak -f` does seem a bit risky does it not?

Reply:
The command is only executed by the maintainer when he wants to update the keywords.  My workflow has been to find a way to collect the keywords and create a command to see the updates; when I want to check new keywords, I execute the command to fetch, paste the result into the buffer and update the keywords.

Reply:
You could centralise the keywords fetching commands into a script, which we could store into `contrib`. Support several languages even (e.g. Python).

Reply:
If we have multiple of those, then a single script in `contrib` that runs all of them would make it easier for maintainers!
Or maybe putting the script in a comment would be less contentious. When you run it, you want to edit this file anyway, so it's just an `eval %val{selection}` away?